### PR TITLE
Ask feedback to author of community PRs

### PR DESCRIPTION
**Pull request by atomiix**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | This new feature will add a comment with a link to a survey once a PR from the community has been merged to get the author's feedback. It will happen once per month max.
| Type?         | new feature
| BC breaks?    | yes
| Deprecations? | no
| Fixed ticket? | Partialy Fixes https://github.com/PrestaShop/prestashop-retro/issues/67
| How to test?  | Tests should passes

### CODE REVIEW

File: src/AppBundle/PullRequests/Listener.php
Comment (Progi1984) [inline src/AppBundle/PullRequests/Listener.php:22]:
```suggestion
```

File: src/AppBundle/PullRequests/Listener.php
Comment (Progi1984) [inline src/AppBundle/PullRequests/Listener.php:74]: Move to use `use function count`

File: src/AppBundle/PullRequests/Repository.php
Comment (Progi1984) [inline src/AppBundle/PullRequests/Repository.php:159]: Move to use `use function count`

File: src/AppBundle/Search/Repository.php
Comment (Progi1984) [inline src/AppBundle/Search/Repository.php:32]: Isn't it a BC to add this parameter and not define a default value ?

File: src/AppBundle/PullRequests/Listener.php
Comment (atomiix) [reply src/AppBundle/PullRequests/Listener.php:74]: php-cs-fixer won't accept it I already tried ;) Do you think I should change the rule though?

File: src/AppBundle/Search/Repository.php
Comment (atomiix) [reply src/AppBundle/Search/Repository.php:32]: Yeah, you're right but I don't think it matters as we're talking about PrestonBot here? 🤔

Comment (Progi1984) [reply src/AppBundle/Search/Repository.php:32]: Keep always good practices ;)

File: src/AppBundle/PullRequests/Listener.php
Comment (Progi1984) [reply src/AppBundle/PullRequests/Listener.php:74]: Ping @PrestaShop/prestashop-core-developers

Comment (PierreRambaud) [reply src/AppBundle/PullRequests/Listener.php:74]: What is the reason of adding the backslash here? :thinking:

Comment (atomiix) [reply src/AppBundle/PullRequests/Listener.php:74]: To make php-cs-fixer happy, that's one of its rules

File: src/AppBundle/Search/Repository.php
Comment (PierreRambaud) [reply src/AppBundle/Search/Repository.php:32]: We can say it's a BC and increase the version of the tool :wink: 
But since we are not using prestonbot as a library or a pluggable application it's not a problem, we can break whatever we want :D

**Comment by matks** [comment]
@atomiix what is the status of this PR ? still valid ? or stale ?

**Comment by atomiix** [comment]
@matks still valid!

**Comment by Progi1984** [comment]
> @matks still valid!

And review too ;)

File: src/AppBundle/PullRequests/Listener.php
Comment (atomiix) [reply src/AppBundle/PullRequests/Listener.php:28]: I did it on purpose to make it more readable by separating Twig-related const  and other const.

File: src/AppBundle/PullRequests/Repository.php
Comment (atomiix) [reply src/AppBundle/PullRequests/Repository.php:159]: I can't do it because of php-cs-fixer rules

**Comment by LouiseBonnard** [comment]
Hi @atomiix, is there some work left to do before merging this PR?

**Comment by atomiix** [comment]
@LouiseBonnard I had to rebase it. Now waiting for reviews ;)

**Review by Progi1984 — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

File: src/AppBundle/PullRequests/Repository.php
Comment (PierreRambaud) [inline src/AppBundle/PullRequests/Repository.php:183]: This kind of string should be store in a file :thinking:  
(Can be done in a new PR)

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
@atomiix Need a rebase

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thanx @atomiix !

**Pull request merged**
